tests/unit/routes/test_feature.py

```diff
@@ -471,7 +471,7 @@ def test_update_200_and_422(self, test_api_client_persistent, create_success_res
             "Discrepancies found between the promoted feature version you are trying to promote to "
             "PRODUCTION_READY, and the input table.\n"
             "{'feature_job_setting': {"
-            "'data_source': FeatureJobSetting(blind_spot='600s', period='1800s', offset='300s', execution_buffer='0s'), "
+            "'data_source': FeatureJobSetting(blind_spot='600s', period='86400s', offset='300s', execution_buffer='0s'), "
```

Comment:
Seems like there is a small side effect (the period of the data source setting may not be equal to the feature job setting of the data source).